Description: Fix FTBFS on hurd due to unset PATH_MAX
 .
 Commited upstream, this patch is to be dropped with 9.11-1.
Author: Emanuele Giaquinta
Origin: upstream
Bug-Debian: http://bugs.debian.org/624736
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Updated: 2011-05-01

Index: pkg-urxvt/src/rxvt.h
===================================================================
--- pkg-urxvt.orig/src/rxvt.h	2011-05-01 10:38:55.507910555 -0400
+++ pkg-urxvt/src/rxvt.h	2011-05-01 10:38:59.407929902 -0400
@@ -119,10 +119,6 @@
 # define EXIT_FAILURE           1       /* exit function failure */
 #endif
 
-#ifndef PATH_MAX
-# define PATH_MAX 16384
-#endif
-
 /****************************************************************************/
 
 // exception thrown on fatal (per-instance) errors
Index: pkg-urxvt/src/rxvtc.C
===================================================================
--- pkg-urxvt.orig/src/rxvtc.C	2011-05-01 10:38:55.547910756 -0400
+++ pkg-urxvt/src/rxvtc.C	2011-05-01 10:38:59.407929902 -0400
@@ -33,8 +33,6 @@
 #include "rxvtdaemon.h"
 #include "libptytty.h"
 
-#include "rxvt.h"
-
 #define STATUS_SUCCESS           0
 #define STATUS_FAILURE           1
 #define STATUS_CONNECTION_FAILED 2
Index: pkg-urxvt/src/rxvtdaemon.C
===================================================================
--- pkg-urxvt.orig/src/rxvtdaemon.C	2011-05-01 10:38:55.595910994 -0400
+++ pkg-urxvt/src/rxvtdaemon.C	2011-05-01 10:38:59.411929916 -0400
@@ -29,7 +29,6 @@
 #include <errno.h>
 #include <sys/types.h>
 #include <sys/utsname.h>
-#include <limits.h>
 
 #include "rxvtdaemon.h"
 
Index: pkg-urxvt/src/rxvtdaemon.h
===================================================================
--- pkg-urxvt.orig/src/rxvtdaemon.h	2011-05-01 10:38:55.639911213 -0400
+++ pkg-urxvt/src/rxvtdaemon.h	2011-05-01 10:38:59.411929916 -0400
@@ -1,6 +1,12 @@
 #ifndef RXVT_DAEMON_H
 #define RXVT_DAEMON_H
 
+#include <limits.h>
+
+#ifndef PATH_MAX
+# define PATH_MAX 16384
+#endif
+
 #include "rxvtutil.h"
 
 struct rxvt_connection
